src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,38 @@
+package duke;
+
+import duke.command.*;
+
+/**
+ * deals with making sense of the user command
+ */
+
+public class Parser {
+    public static Command parse(String fullCommand) {
+        String firstWord = fullCommand.split(" ")[0];
+
+        switch (firstWord) {
+            case "list":
+                return new ListCommand(fullCommand);
+            case "done":
+                return new DoneCommand(fullCommand);
+            case "delete":
+                return new DeleteCommand(fullCommand);
+            case "todo":
+                return new ToDoCommand(fullCommand);
+            case "deadline":
+                return new DeadlineCommand(fullCommand);
+            case "event":
+                return new EventCommand(fullCommand);
+            case "find":
+                return new FindCommand(fullCommand);
+            case "clear":
+                return new ClearCommand(fullCommand);
+            case "help":
+                return new HelpCommand(fullCommand);
+            case "bye":
+                return new ExitCommand(fullCommand);
+            default:
+                return new InvalidCommand(fullCommand);
+        }
```

Comment:
Styling error. Consider following the style specifications for the switch-case block? Specifically, the indentation levels for the cases in your code should be 8 and not 12.